scripts/nfs-server-service.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
-cat /lib/systemd/system/nfs-server.service
+# RM installation doc recommends to use nfs-server drop-in conf files
```

Comment:
This same script needs to work on systems configured for 1.1.x using 1.1.x guidelines.
So if CC version is '1.0.x' or '1.1.x', then cat /lib/systemd/system/nfs-server.service otherwise cat /etc/systemd/system/nfs-server.service.d/nfs-server.conf